homeassistant/components/lg_thinq/sensor.py

```diff
@@ -0,0 +1,590 @@
+    ),
+    ThinQProperty.ONE_TOUCH_FILTER: SensorEntityDescription(
+        key=ThinQProperty.ONE_TOUCH_FILTER,
+        translation_key=ThinQProperty.ONE_TOUCH_FILTER,
+    ),
+}
+RUN_STATE_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.CURRENT_STATE: SensorEntityDescription(
+        key=ThinQProperty.CURRENT_STATE,
+        translation_key=ThinQProperty.CURRENT_STATE,
+    ),
+    ThinQProperty.COCK_STATE: SensorEntityDescription(
+        key=ThinQProperty.COCK_STATE,
+        translation_key=ThinQProperty.COCK_STATE,
+    ),
+    ThinQProperty.STERILIZING_STATE: SensorEntityDescription(
+        key=ThinQProperty.STERILIZING_STATE,
+        translation_key=ThinQProperty.STERILIZING_STATE,
+    ),
+    ThinQProperty.GROWTH_MODE: SensorEntityDescription(
+        key=ThinQProperty.GROWTH_MODE,
+        translation_key=ThinQProperty.GROWTH_MODE,
+    ),
+    ThinQProperty.WIND_VOLUME: SensorEntityDescription(
+        key=ThinQProperty.WIND_VOLUME,
+        translation_key=ThinQProperty.WIND_VOLUME,
+    ),
+}
+TEMPERATURE_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.TARGET_TEMPERATURE: SensorEntityDescription(
+        key=ThinQProperty.TARGET_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        translation_key=ThinQProperty.TARGET_TEMPERATURE,
+    ),
+    ThinQProperty.DAY_TARGET_TEMPERATURE: SensorEntityDescription(
+        key=ThinQProperty.DAY_TARGET_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key=ThinQProperty.DAY_TARGET_TEMPERATURE,
+    ),
+    ThinQProperty.NIGHT_TARGET_TEMPERATURE: SensorEntityDescription(
+        key=ThinQProperty.NIGHT_TARGET_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key=ThinQProperty.NIGHT_TARGET_TEMPERATURE,
+    ),
+    ThinQProperty.TEMPERATURE_STATE: SensorEntityDescription(
+        key=ThinQProperty.TEMPERATURE_STATE,
+        translation_key=ThinQProperty.TEMPERATURE_STATE,
+    ),
+    ThinQProperty.CURRENT_TEMPERATURE: SensorEntityDescription(
+        key=ThinQProperty.CURRENT_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key=ThinQProperty.CURRENT_TEMPERATURE,
+    ),
+}
+WATER_FILTER_INFO_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.USED_TIME: SensorEntityDescription(
+        key=ThinQProperty.USED_TIME,
+        native_unit_of_measurement=UnitOfTime.MONTHS,
+        translation_key=ThinQProperty.USED_TIME,
+    ),
+}
+WATER_INFO_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.WATER_TYPE: SensorEntityDescription(
+        key=ThinQProperty.WATER_TYPE,
+        translation_key=ThinQProperty.WATER_TYPE,
+    ),
```

Comment:
What does this do?

Reply:
Water type is as belows

COLD | Cold water
-- | --
HOT | Hot water
NORMAL | Normal water
MINERAL | Mineral water
SODA | Soda water

We have the link in the document.
https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/water-purifier/